documentation/src/docs/asciidoc/docinfos/index-docinfo.html

```diff
@@ -0,0 +1 @@
+<link rel="icon" type="image/png" href="{{ site.baseurl }}/img/junit5-logo.png">
```

Comment:
> As you can see from the screenshot below, {{ site.baseurl }} is not replaced by Jekyll:

I'll try the solution with the junit5-logo.png in the images folder. 

> In addition, even if this did work, the Javadoc pages would not benefit from the current changes.

Ok, so then what would work for javadocs? Locally, this solution worked for javadocs as well.